app/lib/actions/index.js

```diff
@@ -1,7 +1,5 @@
-import { requestSession } from './sessions';
+import rpc from '../rpc';
 
 export function init () {
-  return (dispatch) => {
-    dispatch(requestSession());
-  };
+  rpc.emit('init');
```

Comment:
With my PR it's the same, but the request session creation is relying on createWindow callback: 
in master on `init()` is called `requestSessions()` that fire `rpc.emit('init')` if `const initial = null == sessions.activeUid;`; 
in my PR on `init()` is fired `rpc.emit('init')`, that create the session through the createWindow callback `rpc.emit('session add req')`, so essentially is the same but `initial`check is not needed.

I've done this because a blank session spawned on createWindow is not always the wanted behaviour, like in our case where we want to call `rpc.emit('open file', { path })` in place of a blank session. Otherwise two session would be created on createWindow event.